llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -2068,7 +2068,10 @@ MemoryDepChecker::getDependenceDistanceStrideAndSize(
   // it in the caller isDependent.
```

Comment:
`// If store sizes are not the same, set TypeByteSize to zero ...`

Outdated comment